### PR TITLE
Dockerize inspirehep

### PR DESCRIPTION
**Pull request by ammirate**

### CODE REVIEW

File: inspirehep/wsgi.py [hunk 1]
```diff
@@ -5,7 +5,20 @@
...
+    config_loader=config_loader,
+    blueprint_entry_points=["invenio_base.api_blueprints"],
+    extension_entry_points=["invenio_base.api_apps"],
+    converter_entry_points=["invenio_base.api_converters"],
+    wsgi_factory=create_wsgi_factory({"/api": create_api}),
+    instance_path=instance_path,
+    app_class=app_class(),
+)
+
+application = create_app()
```
Comment (drjova) [inline inspirehep/wsgi.py]: I'd move this to factory.py and reuse the app in ``cli``, ``wsgi`` and ``celery``

File: ui/docker/Dockerfile-ui [hunk 2]
```diff
@@ -0,0 +1,32 @@
+FROM nginx:stable
+
+EXPOSE 8081
+CMD ["nginx", "-g", "daemon off;"]
+ARG CACHE_DATE=not_a_date
+
+ENV REACT_APP_SENTRY_DSN=https://8d6824a9531c437fbb6de2b3062dfa94@sentry.inspirehep.net/9 \
```
Comment (harunurhan) [inline ui/docker/Dockerfile-ui]: you should remove all of this and add `EVN REACT_APP_ENABLE_LOCAL_LOGIN=YES`

File: docker-compose.yml [hunk 3]
```diff
@@ -11,23 +11,62 @@ services:
...
+      - .:/opt/inspire/ui
+  web:
+    extends:
+      file: docker-services.yml
+      service: app
+    ports:
+     - "8000:8000"
+    command: pipenv run gunicorn --reload -b 0.0.0.0:8000  --access-logfile "-" --error-logfile - inspirehep.wsgi:application
+    volumes:
+      - .:/opt/inspire
```
Comment (pazembrz) [inline docker-compose.yml]: To be safe (that docker won't change permissions etc) you should add `ro` (read only) option to the volume
`- .:/opt/inspire:ro`

File: docker-compose.yml [hunk 4]
```diff
@@ -11,23 +11,62 @@ services:
...
+  worker:
+    extends:
+      file: docker-services.yml
+      service: app
+    command: ./scripts/server
+    volumes:
+      - .:/opt/inspire
```
Comment (pazembrz) [inline docker-compose.yml]: also here

**Comment by drjova** [comment]
@ammirate tested locally and works well 👍

**Pull request merged**